index.html

```diff
@@ -0,0 +1,666 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,300;0,700;0,900;1,400&display=swap"
+        rel="stylesheet" />
+    <title>Nigeria Music Concert NMC 2023</title>
+    <link rel="icon" type="image/x-icon"
+        href="https://img.freepik.com/premium-vector/retro-microphone-vintage-background-poster_606304-137.jpg?w=2000">
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+        integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous" />
+
+    <link rel="stylesheet" href="css/main.css" />
+</head>
+
+<body>
+    <div class="container-fluid">
+        <!--upper navigation-->
+
+        <nav>
+            <div class="row top_navigation">
+                <div class="col-9"></div>
+                <div class="col-3">
+                    <ul>
+                        <li>
+                            <img src="images/facebook.png" alt="facebook">
+                        </li>
+                        <li>
+                            <img src="images/twitter.png" alt="twitter">
+                        </li>
+                        <li>English</li>
+                        <li>My page</li>
+                        <li>Logout</li>
+                    </ul>
+                </div>
+                <div class="col-2"></div>
+            </div>
+        </nav>
+
+        <!--Lower navigation-->
+        <nav>
+            <div class="row lower_navigation">
+                <div class="col-1"></div>
+                <div class="col-2 about_logo">
+                    <img src="images/nmc 2.webp" alt="">
+                </div>
+                <div class="col-1"></div>
+                <div class="col-8">
+                    <ul>
+                        <li>
+                            <a href="about.html">About</a>
+                        </li>
+                        <li>
+                            <a href="##">Program</a>
+                        </li>
+                        <li>
+                            <a href="##">Join</a>
+                        </li>
+                        <li>
+                            <a href="##">Sponsor</a>
+                        </li>
+                        <li>
+                            <a href="##">News</a>
+                        </li>
+                        <li>
+                            <a href="##">Follow us</a>
+                        </li>
```

Comment:
[OPTIONAL]
- [ ] It would be nicer if once the user hits a navigation link, they are redirected to that specific section on the page :smile:

Reply:
hmm sounds cool, will add that